generator/osm2type.cpp

```diff
@@ -529,6 +529,8 @@ namespace ftype
           { "bicycle", "!", [&params] { params.AddType(types.Get(CachedTypes::NOBICYCLE)); }},
           { "bicycle", "~", [&params] { params.AddType(types.Get(CachedTypes::YESBICYCLE)); }},
           { "cycleway", "~", [&params] { params.AddType(types.Get(CachedTypes::YESBICYCLE)); }},
+          { "cycleway:right", "~", [&params] { params.AddType(types.Get(CachedTypes::YESBICYCLE)); }},
```

Comment:
@ygorshenin you are right, the wiki states: "It should then be assumed that cycle traffic is allowed to flow in the customary direction for traffic on that side of the road". The same with `cycleway=opposite_lane`, which is even more complicated. But the trouble is, to correctly use this data in routing, we need to start storing and using driving side hints, which we don't have now. Otherwise `cycleway:left=lane` does not tell us in which direction that lane goes.

So I propose to leave this as is. @bykoianko?